src/soc/ibm/power9/istep_14_2.c

```diff
@@ -0,0 +1,51 @@
+#include <cpu/power/istep_14.h>
+#include <cpu/power/istep_13.h>
+
+static void thermalInit(void)
+{
+	for (size_t MCSIndex = 0; MCSIndex < MCS_PER_PROC; ++MCSIndex) {
+		for (size_t MCAIndex = 0; MCAIndex < MCA_PER_MCS; ++MCAIndex) {
+				mca_and_or(
+					mcs_to_nest[mcs_ids[MCSIndex]], MCAIndex, MCA_MBA_FARB3Q,
+					~PPC_BITMASK(0, 45),
+					PPC_BIT(10) | PPC_BIT(25) | PPC_BIT(37));
+		}
+		scom_and_for_chiplet(mcs_ids[MCSIndex], MCS_MCMODE0, PPC_BIT(21));
+	}
+}
+
+static void progMCMode0(chiplet_id_t MCSTTarget)
+{
+	uint64_t l_scomMask =
+		PPC_BIT(MCS_MCMODE0_DISABLE_MC_SYNC)
+		| PPC_BIT(MCS_MCMODE0_DISABLE_MC_PAIR_SYNC);
+	uint64_t l_scomData =
+		PPC_BIT(MCS_MCMODE0_DISABLE_MC_SYNC)
+		| PPC_BIT(MCS_MCMODE0_DISABLE_MC_PAIR_SYNC);
+	scom_and_or_for_chiplet(
+		MCSTTarget, MCS_MCMODE0,
```

Comment:
This also should have `+ 0x80 * MCSIndex` but there is no index available in the function, so probably it would be easiest to pass that index and convert it to chiplet ID inside the function. In any case, `MCSTTarget` is misleading, this is Nest ID.